algorithm/marine/soca_2cice_global.yaml.j2

```diff
@@ -23,23 +23,39 @@ variable change:
       min hice: 0.5
       min hsno: 0.1
   cice background state:
-    restart: {{ ice_rst }}
+    {% if marine_do_iau %}
+    restart: ./Data/{{ marine_window_begin_fv3 }}.cice_model.res.nc
+    {% else %}
+    restart: ./Data/{{ marine_window_middle_fv3 }}.cice_model.res.nc
+    {% endif %}
```

Comment:
Note: we don't do IAU for sea ice (since CICE model doesn't support it), only for MOM6. Because of that, the background here should always be at the beginning of the window. This applies here, and elsewhere in the file for the ice files.
Having said that, that's a neat trick!

Reply:
It's confusing, but here, iau refers to the type of cycling we are doing. We always use iau for mom6.

Reply:
Ah, good point, ignore my previous comment.